<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Chat</class>
 <widget class="QWidget" name="Chat">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>297</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>username</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../Images.qrc">
    <normaloff>:/images/qnetsoul.png</normaloff>:/images/qnetsoul.png</iconset>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="0" column="0">
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="statusLabel">
       <property name="minimumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>20</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="pixmap">
        <pixmap resource="../Images.qrc">:/images/offline.png</pixmap>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLabel" name="loginLabel">
       <property name="text">
        <string>login</string>
       </property>
      </widget>
     </item>
     <item row="0" column="2">
      <spacer name="horizontalSpacer">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item row="0" column="3">
      <widget class="QLabel" name="portraitLabel">
       <property name="minimumSize">
        <size>
         <width>35</width>
         <height>40</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>35</width>
         <height>40</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
       <property name="scaledContents">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="4">
      <widget class="QTextBrowser" name="outputTextBrowser">
       <property name="openExternalLinks">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item row="2" column="0" colspan="4">
      <widget class="InputTextEdit" name="inputTextEdit">
       <property name="minimumSize">
        <size>
         <width>256</width>
         <height>42</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>80</height>
        </size>
       </property>
       <property name="acceptRichText">
        <bool>true</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>InputTextEdit</class>
   <extends>QTextEdit</extends>
   <header>InputTextEdit.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../Images.qrc"/>
  <include location="../Images.qrc"/>
  <include location="../Images.qrc"/>
 </resources>
 <connections/>
</ui>
